Dp-202-shop/server/src/db/models/OrderItem.model.ts

```diff
@@ -7,17 +7,15 @@ import {Order} from "./Order.model";
 export interface OrderItemAttributes {
   quantity: number;
   id?: number;
-  order_id?: number;
-  product_id?: number;
   createdAt?: Date;
   updatedAt?: Date;
 }
 
 interface OrderItemCreationAttributes extends Optional<OrderItemAttributes, 'id'> {}
 
-@Table({ tableName: 'order_items' })
+@Table({ tableName: 'order-items' })
 export class OrderItem extends Model<OrderItemAttributes, OrderItemCreationAttributes> {
-  public static readonly tableName: string = 'order_items';
+  public static readonly tableName: string = 'order-items';
 
```

Comment:
don't delete anything from db without informing. Put it back. It's the fields that are not necessary to enter